README.md

```diff
@@ -1,3 +1,49 @@
-# tsuru-performance-tests
+# Tsuru Performance test suite
+
+The Tsuru test suite is a set of jmeter-ruby-dsl scripts describing a set of performance testing scenarios.
+They each can be found in the `dsl` folder and run locally or exported to jmx files for running on Blazemeter,
+Flood.io or on your own servers, to provide more resources to a performance test.
+
+Currently, these tests are only runnable against the CI environment.
+
+## How to install
+
+```
+Brew install jmeter
+git clone https://github.com/alphagov/tsuru-performance-testing
+cd ./tsuru-performance-testing
+bundle install
+```
+
+## How to run these tests locally
+
+To run the tests:
+
+```
+ruby deploy_apps.rb ruby deploy_apps.rb <TSURU_TARGET_URL> 100 100
```

Comment:
Oh, I've just realised the dependencies aren't listed in `Gemfile`, but they should be because I get this error:
```
➜  dsl git:(98091996-deploy-mulitple-apps-tsuru) b ruby deploy_apps.rb https://dcarley-api.tsuru.paas.alphagov.co.uk/ 100 100
deploy_apps.rb:2:in `require': cannot load such file -- ruby-jmeter (LoadError)
        from deploy_apps.rb:2:in `<main>'
```